cmd/snap-seccomp/main_test.go

```diff
@@ -127,32 +132,29 @@ int main(int argc, char* argv[])
 
 var seccompSyscallRunnerContent = []byte(`
 #define _GNU_SOURCE
+#include <errno.h>
 #include <stdlib.h>
 #include <sys/syscall.h>
 #include <unistd.h>
 int main(int argc, char** argv)
 {
-    int l[7];
+    int l[7], syscall_ret, ret = 0;
     for (int i = 0; i < 7; i++)
         l[i] = atoi(argv[i + 1]);
     // There might be architecture-specific requirements. see "man syscall"
     // for details.
-    syscall(l[0], l[1], l[2], l[3], l[4], l[5], l[6]);
-    syscall(SYS_exit, 0, 0, 0, 0, 0, 0);
+    syscall_ret = syscall(l[0], l[1], l[2], l[3], l[4], l[5], l[6]);
+    if (syscall_ret < 0 && errno == 911) {
+        ret = 10;
+    }
+    syscall(SYS_exit, ret, 0, 0, 0, 0, 0);
     return 0;
 }
 `)
 
-func lastKmsg() string {
-	output, err := exec.Command("dmesg").CombinedOutput()
-	if err != nil {
-		return err.Error()
-	}
-	l := strings.Split(string(output), "\n")
-	return fmt.Sprintf("Showing last 10 lines of dmesg:\n%s", strings.Join(l[len(l)-10:], "\n"))
-}
```

Comment:
To be clear, this is a question for @mvo5 since this change was made in https://github.com/snapcore/snapd/pull/3998/commits/c09d060b8e41d0a90c3bed639d1ce1f41f6e9952